src/protocol/messages2.rs

```diff
@@ -54,8 +54,7 @@ impl Display for TakerToMakerMessage {
             Self::GetOffer(_) => write!(f, "GetOffer"),
             Self::SwapDetails(_) => write!(f, "SwapDetails"),
             Self::SendersContract(_) => write!(f, "SendersContract"),
```

Comment:
To keep it consistent with the `MakerToTakerMessage` should we call it `SendersContractFromTaker`?